mkosi/config.py

```diff
@@ -2879,7 +2892,16 @@ def load_environment(args: argparse.Namespace) -> dict[str, str]:
     if dnf := os.getenv("MKOSI_DNF"):
         env["MKOSI_DNF"] = dnf
 
-    for s in args.environment:
+    environment_values = []
+    for env_file in args.environment_files:
+        with open(env_file) as f:
+            environment_values.extend(f.read().strip().split("\n"))
+    if len(environment_values) != 0:
+        environment_values.extend(args.environment)
+    else:
+        environment_values = args.environment
+
+    for s in environment_values:
         key, sep, value = s.partition("=")
         value = value if sep else os.getenv(key, "")
         env[key] = value
```

Comment:
Now that we support env files we should call `strip()` on both keys and values here to allow users to put whitespace in their env files on lines if they'd like.